include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -57,6 +57,86 @@ template <int dim,
 class FEEvaluation;
 
 
+/**
+ * @brief The EvaluationFlags enum
+ */
+namespace EvaluationFlags
+{
+  /**
+   * @brief The type enum
+   */
+  enum type
+  {
+    nothing   = 0,
+    values    = 0x1,
+    gradients = 0x2,
+    hessians  = 0x4
```

Comment:
It's not a big deal, but could we name these `evaluate_default`, `evaluate_values`, etc. so that they agree with `UpdateFlags`?

Reply:
One thing I would like to avoid is long names that contain redundancies - we already have `EvaluationFlags::` in the name, so it feels superfluous. These names are typed quite often in a matrix-free code.

Reply:
I liked the shorter notation because ``integrate(EvaluationFlags::evaluate_values);`` is verbose. Thoughts?

Reply:
oh, these have a namespace. Sure.